src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -4,16 +4,18 @@
 # --------------------------------------------------------------------------------------------
 
 # pylint: disable=line-too-long
-
+import argparse
+from azure.cli.core.util import CLIError
 from azure.cli.core.commands.validators import get_default_location_from_resource_group
-from azure.cli.core.commands.parameters import resource_group_name_type, get_enum_type
+from azure.cli.core.commands.parameters import resource_group_name_type, get_enum_type, get_three_state_flag
 from azure.cli.core.util import get_json_object
+from azure.cli.command_modules.servicefabric._validators import validate_create_service, validate_update_application, validate_create_application
 
 
 def load_arguments(self, _):  # pylint: disable=too-many-statements
     with self.argument_context('sf') as c:
-        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None, help='The resource group name')
-        c.argument('cluster_name', options_list=['--name', '--cluster-name', '-n'], help='Specify the name of the cluster, if not given it will be same as resource group name')
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None, help='Specify the resource group name. You can configure the default group using `az configure --defaults group=<name>`')
+        c.argument('cluster_name', options_list=['--cluster-name', '-n'], help='Specify the name of the cluster, if not given it will be same as resource group name')
```

Comment:
'-n' or '--name' should be kept for the operation target. It can be a cluster, an application or an application type, which depends on the operation commands.

Reply:
ok changed -n to -c alias for clusters so it can be used for targets